sandbox/rocky/tf/plotter/__init__.py

```diff
@@ -0,0 +1 @@
+from sandbox.rocky.tf.plotter import *
```

Comment:
Please don't use any `*` imports. Every imported symbol should be specified individually. Not all symbols are part of the public API (so those are not imported).

Reply:
Note that PEP8 says that the public API for a module is the symbols listed in the __all__ list defined in the module. We don't enforce this yet, but you can see that the plotter authors actually took advantage. So you should only import APIs from the __all__ list.